api.py

```diff
@@ -111,20 +111,21 @@ async def submit_agent(submission: SubmissionCode, current_user: dict = Depends(
     if not is_agent_safe(submission.code):
         return ErrorResponseModel(status="error", message="Agent code is not safe.")
     
+
     team = database.get_team(session, team_name)
     if not team.league:
         return ErrorResponseModel(status="error", message="Team is not assigned to a league.")
-    
+    ensure_test_league_folder(team.league.game)
     results = run_agent_simulation(submission.code, team.league.game, team_name)
     if not results:
         return ErrorResponseModel(status="error", message="Agent simulation failed.")
     
     try:
         print("team found", team.name, team.league.name, team.league.folder)
         if team.league.folder:
-            league_folder = team.league.folder
+            league_folder = os.path.normpath(team.league.folder).lstrip('\\')
         else:
-            league_folder = f"leagues/user/{team.league.name}"
+            league_folder = f"leagues\\user\\{team.league.name}"
```

Comment:
I don't understand this change. We are hosting on linux, why the backslashes in the file path?